data/hai/hai jobs.txt

```diff
@@ -1714,3 +1714,354 @@ mission "Swiftsong Science Fiction Convention [2]"
+	passengers 2 4 .8
+	deadline
+	to offer
+		random < 5
+	to accept
+		has "outfit (installed): Quantum Keystone"
+	source
+		government "Hai"
+	destination
+		government "Hai"
+		attributes farming
+		distance 4 8
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 10000 300
+		"reputation: Hai" += 3
+		"reputation: Hai (Wormhole Access)" += 3
+		"reputation: Hai Merchant" += 3
+		"reputation: Hai Merchant (Human)" += 3
+		"reputation: Hai Merchant (Sympathizers)" += 3
+		dialog phrase "generic passenger dropoff payment"
+
+mission "Superstitious Hai [3]"
+	name "Superstitious Hai to <planet>"
+	job
+	repeat
+	description "This elderly Hai will pay <payment> for a journey on a keystone-equipped ship to their retirement home on <destination> by <date>. They have requested that their ship have an authentic Hai keystone installed."
+	passengers 1
+	deadline
+	to offer
+		random < 5
+	to accept
+		has "outfit (installed): Quantum Keystone"
+	source
+		government "Hai"
+	destination
+		government "Hai"
+		attributes "retirement"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 10000 300
+		"reputation: Hai" += 3
+		"reputation: Hai (Wormhole Access)" += 3
+		"reputation: Hai Merchant" += 3
+		"reputation: Hai Merchant (Human)" += 3
+		"reputation: Hai Merchant (Sympathizers)" += 3
+		dialog phrase "generic passenger dropoff payment"
+
+
+mission "Superstitious Hai [4]"
+	name "Superstitious Hai to <planet>"
+	job
+	repeat
+	description "This group of <bunks> farm workers will pay <payment> for a journey on a keystone-equipped ship to <destination> by <date>. They have requested that their ship have an authentic Hai keystone installed."
+	passengers 2 4 .8
+	deadline
+	to offer
+		random < 5
+	to accept
+		has "outfit (installed): Quantum Keystone"
+	source
+		government "Hai"
+	destination
+		government "Hai"
+		attributes "farming"
+		distance 4 8
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 10000 300
+		"reputation: Hai" += 3
+		"reputation: Hai (Wormhole Access)" += 3
+		"reputation: Hai Merchant" += 3
+		"reputation: Hai Merchant (Human)" += 3
+		"reputation: Hai Merchant (Sympathizers)" += 3
+		dialog phrase "generic passenger dropoff payment"
+
+
+mission "Superstitious Hai [5]"
+	name "Superstitious Hai to <planet>"
+	job
+	repeat
+	description "A group of elderly Hai will pay <payment> for a vacation to <destination> on a keystone-equipped ship by <date>. They have requested that their ship have an authentic Hai keystone installed."
+	passengers 2 4 .8
+	deadline
+	to offer
+		random < 5
+	to accept
+		has "outfit (installed): Quantum Keystone"
+	source
+		government "Hai"
+	destination
+		government "Hai"
+		attributes "hai vacation"
```

Comment:
I don't think there is any such location.
Did you mean:
```suggestion
		attributes "hai tourism"
```
For reference, the filter:
```
	government "Hai"
	attributes "hai tourism"
```
matches the following planets:

- Darkmetal
- Dustmaker
- Farwater
- Frostmark
- Greenbloom
- Greenview
- Heartvalley
- Makerplace
- Mirrorlake
- Skyfarm
- Snowfeather
- Swiftsong